stream-chat-android-ui-components/build.gradle

```diff
@@ -40,7 +40,6 @@ android {
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
-        coreLibraryDesugaringEnabled true
```

Comment:
@bychkovdmitry Are you asking about this line?

Reply:
yep